### DISCUSSION TITLE
[ help ] It stops in the middle of a "Searching..."

### DISCUSSION THREAD

**Discussion by chikaniki2** [SudhanPlayz/Discord-MusicBot#1062] [Q&A] [answered]
Hi.
I am not good at English.
I use machine translation. Please forgive me.

After executing ">p [youtubeURL]", the movement stops during the "Searching".
It is hosted on glitch, but no error logs are shown.

Other commands such as HELP can be executed normally.

When I enter a Youtube URL and try to play it, after the bot joins the call, it does not proceed from Searching and the bot automatically Disconnects.

The following were implemented

- Remixed with glitch from github

- Set environment variables in .env files
  - Discord_ClientID
  - Discord_ClientSecret
  - Token
  - Spotify_ClientID
  - Spotify_ClientSecret

- LavaLink set to public
  - lavalink.oops.wtf

The bot is online fine, but is there anything else that needs to be set?

That is all.

I am Japanese, and I respect you guys very much!
Thanks for publishing this great app!

**Comment by JotaroKujo0525** [comment, marked as answer]
Hello, please don't host in glitch, second, the reason your bot is just `Searching` is because of your lavalink. Please change your lavalink and don't host in glitch

**Reply by chikaniki2** [reply]
Thanks for the reply!

I changed the host and lava link and it worked!

Thanks for the help!
